Android Gradle build error (Linux host): `Expected output file at [...]/hdr for target hdr but there was none`

Host: Linux x64 (debian-testing based)
Gradle version: 5.1.1
CMake: 3.13.4

The command `./gradlew assembleDebug` fails with:

```
[...]
[12/12] Linking CXX executable third_party/spirv-cross/bin/Debug/armeabi-v7a/spirv-cross
Build instancing armeabi-v7a
ninja: no work to do.
Build push_descriptors armeabi-v7a
ninja: no work to do.
Build afbc armeabi-v7a
ninja: no work to do.
Build terrain_tessellation armeabi-v7a
ninja: no work to do.
Build hello_triangle armeabi-v7a
ninja: no work to do.
Build hdr armeabi-v7a
ninja: no work to do.

> Task :externalNativeBuildDebug FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':externalNativeBuildDebug'.
> Expected output file at /home/hevrard/work/Vulkan-Samples/build/android_gradle/build-native/cmake/debug/armeabi-v7a/samples/api/hdr/hdr for target hdr but there was none
```

Getting rid of "armeabi-v7a" in `build.gradle`'s ndk abiFilters leads to a similar error :

```
Execution failed for task ':externalNativeBuildDebug'.
> Expected output file at /home/hevrard/work/Vulkan-Samples/build/android_gradle/build-native/cmake/debug/arm64-v8a/samples/performance/command_buffer_usage/command_buffer_usage for target command_buffer_usage but there was none
```

--- a/docs/build.md
+++ b/docs/build.md
@@ -131,7 +131,7 @@ adb shell setprop security.perf_harden 0
 
 ## Dependencies
 
-- CMake v3.10+
+- CMake v3.10+ (know to work with 3.10.2)
 - Visual Studio 2017 or above
 - [clang-format-8](#clang-format-and-visual-studio)
 - [CMake Options](#cmake-options)
@@ -172,7 +172,7 @@ build\windows\app\bin\Release\AMD64\vulkan_samples.exe
 
 ## Dependencies
 
-- CMake v3.10+
+- CMake v3.10+ (know to work with 3.10.2)
 - C++14 Compiler (tested on GCC 7.3)
 - [CMake Options](#cmake-options)
 - [3D models](#3d-models)
@@ -205,7 +205,7 @@ cmake --build build/linux --config Release --target vulkan_samples -- -j4
 
 ## Dependencies
 
-- CMake v3.10+
+- CMake v3.10+ (know to work with 3.10.2)
 - Command Line Tools (CLT) for Xcode `xcode-select --install`
 - [Vulkan SDK](https://vulkan.lunarg.com/doc/sdk/latest/mac/getting_started.html) `./install_vulkan.py`
 - [CMake Options](#cmake-options)

--- a/docs/build.md
+++ b/docs/build.md
@@ -131,7 +131,7 @@ adb shell setprop security.perf_harden 0
 
 ## Dependencies
 
-- CMake v3.10+ (know to work with 3.10.2)
+- CMake v3.10+ (known to work with 3.10.2)
 - Visual Studio 2017 or above
 - [clang-format-8](#clang-format-and-visual-studio)
 - [CMake Options](#cmake-options)
@@ -172,7 +172,7 @@ build\windows\app\bin\Release\AMD64\vulkan_samples.exe
 
 ## Dependencies
 
-- CMake v3.10+ (know to work with 3.10.2)
+- CMake v3.10+ (known to work with 3.10.2)
 - C++14 Compiler (tested on GCC 7.3)
 - [CMake Options](#cmake-options)
 - [3D models](#3d-models)
@@ -205,7 +205,7 @@ cmake --build build/linux --config Release --target vulkan_samples -- -j4
 
 ## Dependencies
 
-- CMake v3.10+ (know to work with 3.10.2)
+- CMake v3.10+ (known to work with 3.10.2)
 - Command Line Tools (CLT) for Xcode `xcode-select --install`
 - [Vulkan SDK](https://vulkan.lunarg.com/doc/sdk/latest/mac/getting_started.html) `./install_vulkan.py`
 - [CMake Options](#cmake-options)

